src/sentry/static/sentry/app/components/group/externalIssueForm.jsx

```diff
@@ -119,8 +127,12 @@ class ExternalIssueForm extends AsyncComponent {
       const separator = url.includes('?') ? '&' : '?';
       // We can't use the API client here since the URL is not scoped under the
       // API endpoints (which the client prefixes)
-      const response = await fetch(url + separator + query);
-      return {options: response.ok ? await response.json() : []};
+      try {
+        const response = await fetch(url + separator + query);
+        cb(undefined, {options: response.ok ? await response.json() : []});
```

Comment:
a bit of a nit but `null` might be a more semantically appropriate value here since the argument is defined (in practice, it generally doesn't matter). This is not blocking, so please merge so we don't have to wait on tests.